### PR TITLE
Add Electra

### PR DESCRIPTION
**Pull request by DolphyWind**
This PR adds a language I developed called Electra where you code like an electrician, which is a unique way of writing code. Here's the wiki link: https://esolangs.org/wiki/Electra and here's the GitHub link: https://github.com/DolphyWind/Electra-Lang.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ Curated list of awesome esoteric programming languages, resources and related pr
 * [ChuckScript](https://github.com/angrykoala/chuckscript) - Unary-based language, where the code is a single number.
 * [DNA#](https://esolangs.org/wiki/DNA-Sharp) - Language based on the structure of the DNA.
 * [Dots](https://github.com/josconno/dots) - A turing-complete language using "." symbol only.
+* [Electra](https://github.com/DolphyWind/Electra-Lang) - Electra lets you code like an electrician.
```
Comment (angrykoala) [inline README.md:38]: Avoid reusing the name of the language in the description:
```suggestion
* [Electra](https://github.com/DolphyWind/Electra-Lang) - Code like an electrician.
```

**Review by angrykoala — CHANGES_REQUESTED** [review]
The language looks good, just a minor change in the description

File: README.md (hunk 1 above)
Comment (angrykoala) [reply README.md:38]: I see that the github readme doesn't have any examples, and cannot find the link to the esolang.org page there, consider adding some examples in the readme

**Comment by DolphyWind** [comment]
All done!

**Pull request merged**